dom-creation/main.js

```diff
@@ -0,0 +1,106 @@
+    imageUrl: 'images/charmander.png',
+  },
+  {
+    number: '007',
+    name: 'Squirtle',
+    description:
+      'When it retracts its long neck into its shell, it squirts out water with vigorous force.',
+    imageUrl: 'images/squirtle.png',
+  },
+  {
+    number: '002',
+    name: 'Ivysaur',
+    description:
+      'When the bulb on its back grows large, it appears to lose the ability to stand on its hind legs.',
+    imageUrl: 'images/ivysaur.png',
+  },
+  {
+    number: '005',
+    name: 'Charmeleon',
+    description:
+      'It has a barbaric nature. In battle, it whips its fiery tail around and slashes away with sharp claws.',
+    imageUrl: 'images/charmeleon.png',
+  },
+  {
+    number: '008',
+    name: 'Wartortle',
+    description:
+      'It is recognized as a symbol of longevity. If its shell has algae on it, that Wartortle is very old.',
+    imageUrl: 'images/wartortle.png',
+  },
+  {
+    number: '003',
+    name: 'Venusaur',
+    description:
+      'Its plant blooms when it is absorbing solar energy. It stays on the move to seek sunlight.',
+    imageUrl: 'images/venusaur.png',
+  },
+  {
+    number: '006',
+    name: 'Charizard',
+    description:
+      'It spits fire that is hot enough to melt boulders. It may cause forest fires by blowing flames.',
+    imageUrl: 'images/charizard.png',
+  },
+  {
+    number: '009',
+    name: 'Blastoise',
+    description:
+      'It crushes its foe under its heavy body to cause fainting. In a pinch, it will withdraw inside its shell.',
+    imageUrl: 'images/blastoise.png',
+  },
+];
+
+function renderPokemon(pokemon) {
+  // creating new div and setting class attribute
+  const newDiv = document.createElement('div');
+  newDiv.setAttribute('class', 'column-third');
+  // creating div child, setting class attribute, and adding to new div
+  const innerDiv = document.createElement('div');
+  innerDiv.setAttribute('class', 'pokemon-card');
+  newDiv.appendChild(innerDiv);
+  // creating img element, adding src attribute, and adding it to inner div
+  const newImg = document.createElement('img');
+  newImg.setAttribute('src', pokemon.imageUrl);
+  innerDiv.appendChild(newImg);
+  // creating a new div to be child of inner div
+  const siblingDiv = document.createElement('div');
+  siblingDiv.setAttribute('class', 'pokemon-card-text');
+  innerDiv.appendChild(siblingDiv);
+  // creating h2 for pokemon's name
+  const pokemonName = document.createElement('h2');
+  pokemonName.textContent = pokemon.name;
+  siblingDiv.appendChild(pokemonName);
+  // creating h3 for pokemon's number
+  const pokemonNum = document.createElement('h3');
+  pokemonNum.textContent = pokemon.number;
+  siblingDiv.appendChild(pokemonNum);
+  // creating p for pokemon's description
+  const pokemonDesc = document.createElement('p');
+  pokemonDesc.textContent = pokemon.description;
+  siblingDiv.appendChild(pokemonDesc);
+  // return the created newDiv
+  return newDiv;
+}
```

Comment:
Don't forget the `$` in front of the variable names. You are storing a DOM object, so you should put a `$` before the variable name

Reply:
Thanks Shawn! added and re-pushed just for my future reference.